engines/sci/graphics/screen.cpp

```diff
@@ -218,7 +244,7 @@ GfxScreen::GfxScreen(ResourceManager *resMan) : _resMan(resMan) {
 
 		initGraphics(_displayWidth, _displayHeight + macIconBarBuffer, format);
 	} else
-		initGraphics(_displayWidth, _displayHeight, format);
+		initGraphics(_gfxDrv->screenWidth(), _gfxDrv->screenHeight(), format);
```

Comment:
What would be the expected behaviour if RGB rendering is enabled alongside CGA/EGA/Hercules mode?

Reply:
I have no idea what that option does. I'll give it a try. Maybe it is something that has to be disabled for CGA/Hercules (like the undithering).

Reply:
It does not work with that option enabled. The purpose seems to be improved screen transitions. Since that is probably targeted at VGA games (just guessing here) and won't help anything with CGA and Hercules modes I'll probably disable it.